commit f144df1120ec32a9144f80e34108532d3168c335
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Thu Jan 24 14:03:28 2019 +0000

    char: vcio: Fail probe if rpi_firmware is not found.
    
    Device Tree is now the only supported config mechanism, therefore
    uncomment the block of code that fails the probe if the
    firmware node can't be found.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/char/broadcom/vcio.c b/drivers/char/broadcom/vcio.c
index 72b133e274e0..d2598663a2b5 100644
--- a/drivers/char/broadcom/vcio.c
+++ b/drivers/char/broadcom/vcio.c
@@ -126,10 +126,9 @@ static int __init vcio_init(void)
 
 	np = of_find_compatible_node(NULL, NULL,
 				     "raspberrypi,bcm2835-firmware");
-/* Uncomment this when we only boot with Device Tree
 	if (!of_device_is_available(np))
 		return -ENODEV;
-*/
+
 	vcio.fw = rpi_firmware_get(np);
 	if (!vcio.fw)
 		return -ENODEV;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1467_f144df1120ec32a9144f80e34108532d3168c335.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1467_f144df1120ec32a9144f80e34108532d3168c335.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
